pages/apis/rest_api/agents.md.erb

```diff
@@ -4,7 +4,7 @@
 
 ## List agents
 
-Returns a [paginated list](<%= paginated_resource_docs_url %>) of an organization’s agents.
+Returns a [paginated list](<%= paginated_resource_docs_url %>) of an organization’s agents. The list only includes running agents - agents in a disconnected state are not returned.
```

Comment:
Hey @yob, apologies for resurrecting a merged PR.

Do you think 

> The list only includes connected agents - agents in a disconnected state are not returned.

makes more sense, as it includes connected but idle agents?

Reply:
yup, that reads much better. Technically agents in a "stopping" state will also be returned, but they're not in that state for very long and if you squint it's almost the same as connected. For this purpose at least.

I'll open a revised PR.